package.json

```diff
@@ -32,6 +32,7 @@
     "next": "^12.0.7",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-linkify": "^1.0.0-alpha",
```

Comment:
Shame that the package is abandoned looking its [version history](https://www.npmjs.com/package/react-linkify?activeTab=versions). I guess we can tolerate this for now, but it might be worth it to monitor some alternatives. I don’t have any on top of my head.